gum/backend-darwin/gumcodesegment-darwin.c

```diff
@@ -298,6 +306,29 @@ gum_code_segment_try_map (GumCodeSegment * self,
   return result != MAP_FAILED;
 }
 
+static gboolean
+gum_code_segment_try_remap (GumCodeSegment * self,
+                            gsize source_offset,
+                            gsize source_size,
+                            gpointer target_address)
+{
+  mach_vm_address_t address = (mach_vm_address_t) target_address;
+  mach_port_t task_self = mach_task_self ();
+  vm_prot_t cur_protection, max_protection;
+  mach_vm_address_t source_address;
+
+  source_address = (mach_vm_address_t) self->data + source_offset;
+
+  mach_vm_protect (task_self, source_address, source_size, FALSE,
+      PROT_READ | PROT_EXEC);
+  kern_return_t kr = mach_vm_remap (task_self, &address, source_size, 0,
+      VM_FLAGS_OVERWRITE | VM_FLAGS_FIXED, task_self,
+      (vm_offset_t) (source_address),
```

Comment:
- Redundant parentheses around `source_address`.
- Looks like this code could be re-flown a bit to use more of the horizontal space.